packages/data-stores/src/launch/selectors.ts

```diff
@@ -50,7 +50,7 @@ export const isSelectedPlanPaid = ( state: State ): boolean =>
  *
  * @param state State
  */
-export const hasSelectedDomain = ( state: State ): boolean =>
+export const hasSelectedDomainOrSubdomain = ( state: State ): boolean =>
```

Comment:
`packages/launch/src/focused-launch/index.tsx` needs refactoring too, otherwise it breaks the build

Reply:
Oops, sorry! I've forgotten to commit the second part of the refactoring 🙈
Fixed now. The remaining failing e2e is unrelated.